src/Query/User/Model/Organization.php

```diff
@@ -93,4 +95,9 @@ public function hasAnonymousAccess(): bool
     {
         return $this->hasAnonymousAccess;
     }
+
+    public function isSecurityScanEnabled(): bool
```

Comment:
Could all methods/properties referring security scan be unified?

Reply:
I can't follow. What do you mean by "unified"? Can you give a better example?

Reply:
Mix of `is` and `has`